wear/src/main/res/values-round/strings.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="hello_world">Hello Round World!</string>
```

Comment:
what is this for?

Reply:
This is automatically generated when adding a wear module.

Reply:
Alright, we should clean up useless code